features/src/main/feature/camel-features.xml

```diff
@@ -357,12 +357,14 @@
 <!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.aws-java-sdk2/${aws-sdk-v2-bundle-version}</bundle>-->
 <!--        <bundle>mvn:org.apache.camel/camel-aws2-sns/${project.version}</bundle>-->
 <!--    </feature>-->
-<!--    <feature name='camel-aws2-sqs' version='${project.version}' start-level='50'>-->
-<!--        <feature version='${project.version}'>camel-core</feature>-->
-<!--        <bundle dependency='true'>mvn:commons-io/commons-io/${commons-io-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.aws-java-sdk2/${aws-sdk-v2-bundle-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-aws2-sqs/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-aws2-sqs' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature>awssdk</feature>
+        <bundle dependency="true">mvn:org.apache.camel.karaf/camel-cloudevents/${project.version}</bundle>
+        <bundle dependency='true'>mvn:commons-io/commons-io/2.14.0</bundle>
```

Comment:
Not the expected version of commons-io https://github.com/apache/camel/blob/camel-4.4.x/parent/pom.xml#L105

Reply:
thanks, changed it to 2.15.1